<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>He Hui</author>
 <class>DatabaseConnecterDialogUI</class>
 <widget class="QDialog" name="DatabaseConnecterDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Connecter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="networkDatabasePage">
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="networkDatabaseGroupbox">
         <property name="title">
          <string>Network Database</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="databaseNameLabel">
            <property name="text">
             <string>&amp;Database:</string>
            </property>
            <property name="buddy">
             <cstring>databaseNameEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="databaseNameEdit">
            <property name="toolTip">
             <string>Database Name</string>
            </property>
            <property name="statusTip">
             <string>Database Name</string>
            </property>
            <property name="whatsThis">
             <string>Database Name</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="driverLabel">
            <property name="text">
             <string>D&amp;river:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>driverCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QComboBox" name="driverCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Database Driver</string>
            </property>
            <property name="statusTip">
             <string>Database Driver</string>
            </property>
            <property name="whatsThis">
             <string>Database Driver</string>
            </property>
            <item>
             <property name="text">
              <string>Auto Select</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="hostNameLabel">
            <property name="text">
             <string>&amp;Hostname:</string>
            </property>
            <property name="buddy">
             <cstring>hostnameEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="hostnameEdit">
            <property name="toolTip">
             <string>Host Name Or IP Address Of The Database Server</string>
            </property>
            <property name="statusTip">
             <string>Host Name Or IP Address Of The Database Server</string>
            </property>
            <property name="whatsThis">
             <string>Host Name Or IP Address Of The Database Server</string>
            </property>
            <property name="text">
             <string notr="true">localhost</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="portLabel">
            <property name="text">
             <string>Port:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>portSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="portSpinBox">
            <property name="toolTip">
             <string>Port Of The Database Server</string>
            </property>
            <property name="statusTip">
             <string>Port Of The Database Server</string>
            </property>
            <property name="whatsThis">
             <string>Port Of The Database Server</string>
            </property>
            <property name="specialValueText">
             <string>Default</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="userNameLabel">
            <property name="text">
             <string>&amp;Username:</string>
            </property>
            <property name="buddy">
             <cstring>userNameEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="userNameEdit">
            <property name="toolTip">
             <string>User Name Of The Database Server</string>
            </property>
            <property name="statusTip">
             <string>User Name Of The Database Server</string>
            </property>
            <property name="whatsThis">
             <string>User Name Of The Database Server</string>
            </property>
            <property name="text">
             <string notr="true">root</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="passwordLabel">
            <property name="text">
             <string>&amp;Password:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>passwordEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="passwordEdit">
            <property name="toolTip">
             <string>Password</string>
            </property>
            <property name="statusTip">
             <string>Password</string>
            </property>
            <property name="whatsThis">
             <string>Password</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="localDatabaseFilePage">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="localDatabaseGroupbox">
         <property name="title">
          <string>Local Database File</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>8</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="1" column="0" colspan="3">
           <widget class="QComboBox" name="databaseFilePathComboBox">
            <property name="toolTip">
             <string>Database File Path</string>
            </property>
            <property name="statusTip">
             <string>Database File Path</string>
            </property>
            <property name="whatsThis">
             <string>Database File Path</string>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="browseButton">
            <property name="toolTip">
             <string>Click To Select The Database File</string>
            </property>
            <property name="statusTip">
             <string>Click To Select The Database File</string>
            </property>
            <property name="whatsThis">
             <string>Click To Select The Database File</string>
            </property>
            <property name="text">
             <string>&amp;Browse</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="databaseFilePathLabel">
            <property name="text">
             <string>Database File Path:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="databaseTypeLabel">
       <property name="text">
        <string>Database &amp;Type:</string>
       </property>
       <property name="buddy">
        <cstring>databaseTypeComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="databaseTypeComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Database Type</string>
       </property>
       <property name="statusTip">
        <string>Database Type</string>
       </property>
       <property name="whatsThis">
        <string>Database Type</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Return</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>hostnameEdit</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>userNameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>databaseNameEdit</tabstop>
  <tabstop>driverCombo</tabstop>
  <tabstop>databaseTypeComboBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>databaseFilePathComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
